en/editor/project/index.md

```diff
@@ -6,10 +6,26 @@ The **Project Settings** panel is opened by clicking on **Projects -> Project Se
 
 ## Macro Config
 
-For specific information and code about the engine macro module, you can refer to **Engine Macro** ([GitHub](https://github.com/cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824) | [Gitee](https://gitee.com/mirrors_cocos-creator/engine/blob/3d/cocos/core/platform/macro.ts#L824)), which provides a shortcut to modify the macro configuration, the configured macro will take effect when previewing, building The configured macro will take effect on preview, build, and will also follow the custom engine configuration to update the default values of the current macro configuration.
+**Macro Config** provides a shortcut to modify the macro configuration, the configured macro will take effect when previewing, building The configured macro will take effect on preview, build, and will also follow the custom engine configuration to update the default values of the current macro configuration.
+
+- **ENABLE_TILEDMAP_CULLING**: whether or not enabled tiled map auto culling. If you set the TiledMap skew or rotation, then need to manually disable this, otherwise, the rendering will be wrong.
```

Comment:
```suggestion
- **ENABLE_TILEDMAP_CULLING**: whether or not tiled map auto culling is enabled. If setting the TiledMap skew or rotation, then it is necessary to manually disable this setting, otherwise, the rendering will be wrong.
```